src/AlgoliaSearch.js

```diff
@@ -118,54 +119,56 @@ AlgoliaSearch.prototype = {
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  deleteIndex: function(indexName, callback) {
-    return this._jsonRequest({ method: 'DELETE',
-              url: '/1/indexes/' + encodeURIComponent(indexName),
-              hostType: 'write',
-              callback: callback });
+  deleteIndex: function deleteIndex(indexName, callback) {
```

Comment:
Yes in our eslint-config I have set it as a warning only since it requires some good rewrite and again not everyone likes to do this.

We can set this as an error if we want, still not convinced even if I did it, mostly because it was a warning.

There's no clear benefit of doing it, most recent browsers will show the property name as the function name, so we might even disable it.

what do you think?